### PR TITLE
DS-4291 Properly show results for 0-9 link in browse

### PR DESCRIPTION
**Pull request by aprilherron**
When browsing by Author, if you click the "0-9" link at the top, it is only showing results for 0 instead of 0-9.

Currently, only a facet query prefix for "0" is being used, so no other results are shown. Because facet query prefixes don't support ranges, the simplest solution is to perform separate queries and combine the results for 0-9. This fix performs the additional queries and combines their results and totals.

This is the 6.x version of #2696

### CODE REVIEW

**Review by kshepherd — APPROVED** [review]
+1 by inspection, looks good

File: dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/ConfigurableBrowse.java [hunk 1]
```diff
@@ -856,7 +855,21 @@ private BrowseInfo getBrowseInfo() throws SQLException, UIException
         {
             // Create a new browse engine, and perform the browse
             BrowseEngine be = new BrowseEngine(context);
-            this.browseInfo = be.browse(params.scope);
+            if ("0".equals(params.scope.getStartsWith())) {
+                BrowseInfo info = be.browse(params.scope);
+                for (String num : new String[]{"1","2","3","4","5","6","7","8","9"}) {
+                    // Append our results to info and upate total
```
Comment (nwoodward) [inline dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/ConfigurableBrowse.java]: upate => update

**Review by nwoodward — APPROVED** [review]
:+1: Tested and it works.

**Comment by KingKrimmson** [comment]
My previous implementations was missing some fixes we had made for a client. This new implementation supports both asc/desc orders and reports proper counts as well

**Review by nwoodward — APPROVED** [review]
:+1: Tested the new implementation, and it works, including ordering and counts.

**Pull request merged**